site/en/docs/extensions/mv3/tut_debugging/index.md

```diff
@@ -3,146 +3,285 @@ layout: "layouts/doc-post.njk"
-all** button in the upper right hand corner, then reload the extension.
+Refresh and click the **Errors** button to view the error log. The first error will let you know that the service worker failed to register. This means something went wrong during initiation. 
 
-### Popup {: #debug_popup }
+<figure>
+  {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/5Vq0jbo4AGUp1P9MoU3v.png", alt="ALT_TEXT_HERE", width="800", height="418" %}
+  <figcaption>
+  Service worker registration error message.  
+  </figcaption>
+</figure>
 
-Now that the extension initializes correctly, other components can be tested. Refresh this page, or
-open a new tab and navigate to any page on developer.chrome.com, open the popup and click the green
-square. And... nothing happens.
+{% Aside %}
 
-Navigate back to the Extensions Management Page, the **Errors** button has reappeared. Click it to
-view the new log.
+If the service worker fails to register, you will not be able to access the Chrome DevTools until you fix the registration bug. 
 
-`Uncaught ReferenceError: tabs is not defined`
+{% endAside %}
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/ygCnLz4Dst4mktOQhj4S.png",
-       alt="Extensions Management Page displaying popup error", height="559", width="642" %}
+The second is a is a TypeError:
 
-Popup errors can also be viewed by inspecting the popup.
+<figure>
+  {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/LauDnPFrNyExahWz3RF1.png", alt="ALT_TEXT_HERE", width="800", height="528" %}
+  <figcaption>
+  Uncaught TypeError message.
+  </figcaption>
+</figure>
 
-{% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/PTfS6FEsE6vkbtDVMcsM.png",
-       alt="DevTools displaying popup error", height="189", width="499" %}
+Update the code to reflect the correct call, click the **Clear all** button in the upper right-hand corner, then reload the extension.
 
-The error, `tabs is undefined`, says the extension doesn't know where to inject the content script.
-This can be corrected by calling the [`tabs.query()`][4] method, then selecting the active tab.
+#### Check the service worker status {: #sw-status }
 
-```js/8-12
-  let changeColor = document.getElementById('changeColor');
+You can identify when the service worker wakes up to perform tasks by following these steps:
 
-  chrome.storage.sync.get(['color'], ({color}) => {
-    changeColor.style.backgroundColor = color;
-    changeColor.setAttribute('value', color);
-  });
+1. Open your manifest file in the browser. For example:
+    ```text
+    chrome-extension://EXTENSION_ID/manifest.json
+    ``` 
+2. Inspect the file.
+3. Navigate to the **Application** panel.
+4. Go to the **Service worker** pane.
+
+You can start or stop the service worker using the links next to the **Status** to test
+your code.
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/i1w015KXe7EzN3chRojv.png", alt="Service worker status in the Application panel", width="800", height="453", class="screenshot" %}
```

Comment:
The screenshot is too small, maybe add the option to click to enlarge/

Reply:
Added enlarged versions to both. I'm waiting for the [staged link](https://pr-6072-static-dot-dcc-staging.uc.r.appspot.com/docs/extensions/mv3/tut_debugging/#sw-status) to update. It's slow... 🐌